src/utils/utils.jl

```diff
@@ -46,78 +10,14 @@ function check_column(df::DataFrame, names_array::Array{Symbol,1})
-end
-
-"""
-    getindex(variable::OptVariable,index_set::Array)
-Get the variable data from the specific Scenario by indicating the `var_name` e.g. "COST" and the `index_set` like `[:;"EUR";"pv"]`
-"""
-function get_cep_variable_value(variable::OptVariable,
-                                index_set::Array
-                                )
-                                @warn "get_cep_variable_value deprecated"
-    index_num=[]
-    for i in  1:length(index_set)
-        if index_set[i]==Colon()
-            push!(index_num,Colon())
-        elseif typeof(index_set[i])==Int || typeof(index_set[i])==UnitRange{Int}
-            push!(index_num,index_set[i])
-        else
-            new_index_num=findfirst(variable.axes[i].==index_set[i])
-            if new_index_num==[]
-                throw(@error("$(index_set[i]) not in indexset #$i of Variable $var_name"))
-            else
-                push!(index_num,new_index_num)
-            end
-        end
-    end
-    return getindex(variable.data,Tuple(index_num)...)
-end
-
-"""
-    get_cep_variable_value(scenario::Scenario,var_name::String,index_set::Array)
-Get the variable data from the specific Scenario by indicating the `var_name` e.g. "COST" and the `index_set` like `[:;"EUR";"pv"]`
-"""
-function get_cep_variable_value(scenario::Scenario,
-                                var_name::String,
-                                index_set::Array
-                                )
-                                @warn "get_cep_variable_value deprecated"
-    return get_cep_variable_value(scenario.opt_res.variables[var_name], index_set)
-end
-
-"""
-    get_cep_variable_set(variable::OptVariable,num_index_set::Int)
-Get the variable set from the specific variable and the `num_index_set` like 1
-"""
-function get_cep_variable_set(variable::OptVariable,
-                              num_index_set::Int
-                              )
-                              @warn "get_cep_variable_set deprecated"
-    return variable.axes[num_index_set]
+#Use getindex to return all rows of the DataFrame fulfilling that in column `col` value `val` is found
+function Base.getindex(df::DataFrame, col_and_val::Tuple{Symbol,Any})
+    return df[findall(df[!,col_and_val[1]].==col_and_val[2]), :]
 end
 
-"""
-    get_cep_variable_set(scenario::Scenario,var_name::String,num_index_set::Int)
-Get the variable set from the specific Scenario by indicating the `var_name` e.g. "COST" and the `num_index_set` like 1
-"""
-function get_cep_variable_set(scenario::Scenario,
-                              var_name::String,
-                              num_index_set::Int
-                              )
-                              @warn "get_cep_variable_set deprecated"
-    return  get_cep_variable_set(scenario.opt_res.variables[var_name], num_index_set)
+#Use getindex to return all rows of the DataFrame fulfilling that in column `col` value `val` is found and the column `colon_ind`
```

Comment:
Same here, what does this do in terms of intuition (what is col, what is val)?

Reply:
Additionally: Is this referred to in the documentation (if it is a nice function to know)?